pkg/networking/networking.go

```diff
@@ -37,23 +43,25 @@ type NetworkAccess interface {
 
 // networkImpl is the default implementation of the NetworkAccess interface.
 type networkImpl struct {
-	config       configuration.Configuration
-	staticHeader http.Header
-	proxy        func(req *http.Request) (*url.URL, error)
-	caPool       *x509.CertPool
-	logger       *zerolog.Logger
+	config             configuration.Configuration
+	staticHeader       http.Header
+	proxy              func(req *http.Request) (*url.URL, error)
+	caPool             *x509.CertPool
+	logger             *zerolog.Logger
+	snykAppEnvironment *middleware.UserAgentInfo
 }
 
-// customRoundTripper is a custom http.RoundTripper which decorates the request with default headers.
-type customRoundTripper struct {
+// DefaultHeadersRoundTripper is a custom http.RoundTripper which decorates the request with default headers.
+type DefaultHeadersRoundTripper struct {
```

Comment:
why is this exported now? why the name change?

Reply:
Un-exporting it 👍 

I believe it was called "customRoundTripper" when it was the only RT, but now we have several and they are all "custom". 

Essentially it's an RT that adds the default headers, so I named it "defaultHeadersRoundTripper"